remote-config.jsonc

```diff
@@ -153,7 +153,10 @@
           "message": "DDEV has support for many database browsers. See them all at https://ddev.readthedocs.io/en/stable/users/usage/database-management/#database-guis"
         },
         {
-          "message": "Pro tip: `ddev config --auto` will update your project config to ensure it’s compatible with the version of DDEV that’s running."
+          "message": "Pro tip: `ddev config --auto` will update your project config to ensure it’s compatible with the version of DDEV that’s running, but it won't change configured values."
+        },
+        {
+          "message": "The new `ddev config --update` will change your project config where possible based ont he code that is discovered in your project."
```

Comment:
```suggestion
          "message": "The new `ddev config --update` will change your project config where possible based on the code that is discovered in your project."
```

Reply:
I just don't think we can solve everything in one sentence.